client/lib/safe-image-url/index.js

```diff
@@ -51,22 +58,34 @@ export default function safeImageUrl( url ) {
 		return url;
 	}
 
-	const { hostname, pathname, search } = getUrlParts( url );
+	const parsedUrl = getUrlParts( url );
+	if ( ! parsedUrl.protocol ) {
+		return url;
```

Comment:
Is the intention here to allow relative URLs through? Could this potentially miss a protocol-relative image URL like `//mybadsite.com/badimage.jpg`?

Reply:
I've removed it.